src/data.jl

```diff
@@ -501,6 +501,24 @@ end
 # Getting an index produces a tuple of the energy, 
 Base.getindex(d::DensityOfStates, ind) = (d.energy[ind], d.dos[ind], d.int[ind])
 
+# Setting up for gaussian smearing of DOS curve
+# Fourier transformed gaussian for convolution in smearing
+function Gaussian(
+    sigma::Real,
+    k::Real
+)
```

Comment:
You can also place all function arguments on a single line if the signature is short, but this isn't required.